.github/actions/bump-version/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
This entire action was lifted from work @jhamon did previously: https://github.com/pinecone-io/pinecone-python-client/pull/178

I'll call out lines I modified, which should be within `action.js` and `index.js` where I got rid of modifying local files via `fs.readFileSync` and `fs.writeFileSync`. I'm doing the file manipulating using `sed` in the `release.yaml` file.